lib/library_test_utils.go

```diff
@@ -717,6 +723,53 @@ func testAccountSelect(t *testing.T) bool { //nolint: gocyclo
 	return true
 }
 
+func testLoginWithKeycard(t *testing.T) bool { //nolint: gocyclo
```

Comment:
I recommend putting this unit test in a file that contains tests for `StatusBackend`. `library.go` just calls `StatusBackend` method so here we should only test data serialization.

Reply:
@adambabik I added `TestBackendInjectChatAccount` in the backend tests. I also left this one in lib so that all the functions are tested in the same way, since we do the same for `Login`

Reply:
Hm. Ok, being consistent is actually better. We should refactor it at some point because when we introduce gomobile, we will need to have all logic in `StatusBackend` and tested there as well.